modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementFailureTest.java

```diff
@@ -237,7 +237,7 @@ public void testAddDiscoveryAttributesToNonExistingOrganization() throws IOExcep
         String requestBody = readResource("add-discovery-attributes-request-body.json");
         requestBody = requestBody.replace("${organizationID}", invalidOrganizationID);
         Response response = getResponseOfPostWithOAuth2(endpointURL, requestBody, m2mToken);
-        validateErrorResponseWithoutTraceId(response, HttpStatus.SC_NOT_FOUND, "ORG-60015",
+        validateErrorResponse(response, HttpStatus.SC_NOT_FOUND, "ORG-60015",
                                         invalidOrganizationID);
```

Comment:
I think we can get invalidOrganizationID to same line now if it does not exceed 120 characters

Reply:
Check similar places

Reply:
Done